packages/cli/src/create/helpers/create-contract.ts

```diff
@@ -164,6 +185,27 @@ export async function createContractProject({
         // Write the base contract to the new file
         await writeFile(contractFile, baseContractText);
       }
+
+      if(!baseContract && createExtension) {
+
+        const extensionBoilerplate = readExtensionBoilerPlate(contractObjectName);
+  
+        // Set the contents of the Contract.sol file to the extension contract storage library boilerplate
+        let contractFile = "";
+        contractFile = path.join(root, contractName);
+        if (framework === "hardhat") {
+          fs.unlinkSync(path.join(root, "contracts", "Contract.sol"));
+          contractFile = path.join(root, "contracts", contractName);
+        }
+        if (framework === "forge") {
+          fs.unlinkSync(path.join(root, "src", "Contract.sol"));
```

Comment:
Does the file have to be called `Contract.sol` ? can't we use the extension name entered?

Reply:
We do use the extension name entered 👍🏽